.github/workflows/automation-ondemand-tests.yml

```diff
@@ -104,6 +114,7 @@ jobs:
         with:
           ref: ${{ github.head_ref || github.ref_name }}
       - name: Build Test Image
+        if: inputs.enableChaos || inputs.enableReorg
```

Comment:
does this mean that the test image will be created only when chaos or reorg are enabled? Is that intended?

Reply:
yes that is right. the test image is needed only for running chaos and reorg test which are tests on k8s. The upgrade tests run on docker within the github runner, so no need of test image.